cmd/layers.go

```diff
@@ -157,12 +176,14 @@ func init() {
 
 	layersNonReproducibleCmd.Flags().StringVarP(&rewritesFilepath, "rewrites", "", "", "A JSON file containing a list of path rewrites. Each element of the list is a JSON object with the attributes path, regex and repl: for a given path, the regex is replaced by repl.")
 	layersNonReproducibleCmd.Flags().StringVarP(&permsFilepath, "perms", "", "", "A JSON file containing file permissions")
+	layersNonReproducibleCmd.Flags().StringVarP(&metadataFilepath, "metadata", "", "", "A JSON file containing layer metadata")
```

Comment:
I think we could call this `history`, since it contains the data to set the `History` attribute of an image.
(We could then have to rename `metadata` to `history` in all our Go code. )